app/logic/emailHandler.py

```diff
@@ -71,8 +70,11 @@ def fetch_event_programs(self, program_id):
         """ Fetches all the programs of a particular event """
         # Non-student-led programs have "Unknown" as their id
         if program_id == 'Unknown' or program_id is None:
-            programEvents = ProgramEvent.select(ProgramEvent.program).where(ProgramEvent.event==self.event.id)
-            return [program.program for program in programEvents.objects()]
+            
+            programEvents = Event.select(Event.program_id).where(Event.id==self.event.id)
+            
+            
+            return [program.program_id for program in programEvents.objects()] 
         else:
             return [Program.get_by_id(program_id)]
```

Comment:
This whole function is irrelevant now. It is getting either a program or a list of programs for an event, but events now only have one program (or none). So it should return the Program for the passed in `program_id`, or `event.program`

Reply:
In an attempt to resolve this, we have resorted to returning "self.event.program", and we have scraped the query.